src/Microsoft.Azure.WebJobs.Logging/IFunctionDefinition.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.WindowsAzure.Storage.Table;
+using System;
+using System.Globalization;
+
+namespace Microsoft.Azure.WebJobs.Logging
+{
+    /// <summary>
+    /// Information about a function definition. 
+    /// </summary>
+    public interface IFunctionDefinition
```

Comment:
Agreed. Although this library should be the only thing implementing the interface, so that should mitigate some of it. 

---
In reply to: [57636453](https://github.com/Azure/azure-webjobs-sdk/pull/678#discussion_r57636453) [](ancestors = 57636453)